.github/workflows/ci.yaml

```diff
@@ -9,6 +9,38 @@ env:
   DOCKER_BUILDX_PLATFORMS: linux/amd64,linux/arm64
 
 jobs:
+
+  changes:
```

Comment:
If i understand whats going on here, this `changes` section lets us then conditionally run other stages based on the files that changed? It might be worth a comment and small explanation here since the name wasn't super obvious